.circleci/config.yml

```diff
@@ -0,0 +1,100 @@
+version: 2
+jobs:
+  relay:
+    working_directory: ~/artsy/reaction
+    docker:
+    - image: circleci/node:8
```

Comment:
The image field here is what specifies what version of node we'll be running the tests against. Currently just specified as the latest version of Node 8.